lua/code-link/link.lua

```diff
@@ -9,6 +9,7 @@ end
 
 function CodeLink.new(config)
 	local origin = CodeLink.get_git_origin(config)
+    origin = CodeLink.convert_git_origin(origin)
```

Comment:
```suggestion
	local origin = CodeLink.convert_git_origin(CodeLink.get_git_origin(config))
```
I treat the "origin" as a constant, it would be great if just assigned once